tools/twemoji/generate-shortname-mapping.py

```diff
@@ -14,8 +14,7 @@
             mappings.append((shortname.strip(':'), emoji['emoji']))
 
 # Custom shortcodes
-mappings.append(('slight_smile', '🙂'))
-mappings.append(('slight_frown', '🙁'))
+#mappings.append(('slight_smile', '🙂'))
```

Comment:
:question:

Reply:
I left it in there as documentation, in case we want to add custom shortcodes without having to upstream them first.

(The two that I removed are already upstreamed.)

Reply:
I replaced it with an actual shortcode I just found to be missing :slightly_smiling_face: